manifests/m/mar10/stressor/0.5.1/mar10.stressor.yaml

```diff
@@ -0,0 +1,19 @@
+#
+# Windows Package Manager Manifest
+#
+PackageIdentifier:  mar10.stressor
+PackageVersion:     0.5.1
+PackageLocale:      en-US
+Publisher:          'Martin Wendt'
+PackageName:        'stressor'
+License:            'MIT'
+LicenseUrl:         'https://raw.githubusercontent.com/mar10/stressor/master/LICENSE.txt'
+ShortDescription:   'Stress-test your web app'
+Installers:
+ - Architecture:    x64
+   InstallerType:   msi
+   InstallerUrl:    https://github.com/mar10/stressor/releases/download/v0.5.1/stressor-0.5.1.0-amd64.msi
+   InstallerSha256: 98B70F59583898761CC969C4389350153F8F14F93B8DF9A3C99CA2586DF98CD6
+   ProductCode: "{D4E5F9B7-CF19-455C-859D-7E121A7E4F1C}"
+ManifestType:       singleton
+ManifestVersion: 1.0.0
```

Comment:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
PackageIdentifier: mar10.stressor
PackageVersion: 0.5.1.0
PackageLocale: en-US
Publisher: Martin Wendt
PackageName: stressor
License: MIT
LicenseUrl: https://raw.githubusercontent.com/mar10/stressor/master/LICENSE.txt
ShortDescription: Stress-test your web app
Installers:
- Architecture: x64
  InstallerType: msi
  InstallerUrl: https://github.com/mar10/stressor/releases/download/v0.5.1/stressor-0.5.1.0-amd64.msi
  InstallerSha256: 98B70F59583898761CC969C4389350153F8F14F93B8DF9A3C99CA2586DF98CD6
  ProductCode: "{D4E5F9B7-CF19-455C-859D-7E121A7E4F1C}"
ManifestType: singleton
ManifestVersion: 1.0.0
```

Reply:
Hello,
I understand that `PackageVersion` and path must match.
However, since I use [semantic versioning](https://semver.org) in my project (MAJOR.MINOR.PATCH) I would prefer three digits.
Is this a problem? I noticed that there some other packages using this notation, e.g.

https://github.com/microsoft/winget-pkgs/blob/master/manifests/m/Microsoft/AzureCLI/2.28.0/Microsoft.AzureCLI.yaml